numpy/core/numeric.py

```diff
@@ -1497,10 +1497,12 @@ def rollaxis(a, axis, start=0):
         Input array.
     axis : int
         The axis to roll backwards.  The positions of the other axes do not
-        change relative to one another.
+        change relative to one another. Must be in the range [-n, n), where
```

Comment:
I don't think it's necessary to update the docstring -- nearly every numpy function that takes an axis argument accepts values in this range.